apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/post-content/index.js

```diff
@@ -25,3 +27,20 @@ registerBlockType( 'a8c/post-content', {
 	edit,
 	save,
 } );
+
+const { createHigherOrderComponent } = wp.compose;
+const addContentSlotClassname = createHigherOrderComponent( BlockListBlock => {
+	return props => {
+		if ( props.name !== 'a8c/post-content' ) {
+			return <BlockListBlock { ...props } />;
+		}
+
+		return <BlockListBlock { ...props } className={ 'post-content__block' } />;
+	};
+}, 'addContentSlotClassname' );
```

Comment:
Nice TIL, I didn't know about this one!

Reply:
Nor did I! I found it by just scanning through all the available filters. I think this will come in handy for a couple of other issues. 👍